docs/api.md

```diff
@@ -1552,15 +1557,6 @@ from the `accounts` table.
   
   <!--end-response-body-get-recovery_emails-email-->
 
-##### Error responses
-
-Failing requests may be caused
-by the following errors
-(this is not an exhaustive list):
-
-* `code: 503, errno: 202`:
```

Comment:
@philbooth No more forgetting to update docs 🙌

Reply:
Yeah, although it's mystifying why my commit to reinstate utm params to metrics context didn't update the docs. I don't recall seeing any errors when I committed.

Automation: great when it works, really annoying when it doesn't.